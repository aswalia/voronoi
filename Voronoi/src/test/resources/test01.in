lft:
lft:
lft:
lft:
lft:
point:(7.0,3.0)
point:(6.0,7.0)
point:(5.0,4.0)
point:(5.0,0.0)
point:(2.0,7.0)
lft:
lft:
point:(2.0,4.0)
point:(1.0,0.0)
point:(0.0,5.0)
