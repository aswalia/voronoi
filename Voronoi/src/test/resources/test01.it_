node
rgt-point:(2.0,7.0)
rgt-point:(5.0,0.0)
rgt-point:(5.0,4.0)
node-point:(7.0,3.0) (6.0,7.0)
rgt-point:(0.0,5.0)
node-point:(2.0,4.0) (1.0,0.0)
