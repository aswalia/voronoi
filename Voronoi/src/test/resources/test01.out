Vor:
(0.0,7.0) (1.5,5.5)
lft and rgt: (0.0,5.0) (2.0,7.0)
(1.5,5.5) (-0.05555555555555558,2.388888888888889)
lft and rgt: (0.0,5.0) (2.0,4.0)
(-0.05555555555555558,2.388888888888889) (-2.0,2.0)
lft and rgt: (0.0,5.0) (1.0,0.0)
(3.0,1.625) (-0.05555555555555558,2.388888888888889)
lft and rgt: (2.0,4.0) (1.0,0.0)
(3.0,1.625) (3.0,-2.0)
lft and rgt: (1.0,0.0) (5.0,0.0)
(3.5,2.0) (3.0,1.625)
lft and rgt: (2.0,4.0) (5.0,0.0)
(3.5,2.0) (5.25,2.0)
lft and rgt: (5.0,0.0) (5.0,4.0)
(5.25,2.0) (7.5,0.5)
lft and rgt: (5.0,0.0) (7.0,3.0)
(6.785714285714286,5.071428571428571) (5.25,2.0)
lft and rgt: (5.0,4.0) (7.0,3.0)
(6.785714285714286,5.071428571428571) (8.5,5.5)
lft and rgt: (7.0,3.0) (6.0,7.0)
(4.0,6.0) (6.785714285714286,5.071428571428571)
lft and rgt: (5.0,4.0) (6.0,7.0)
(4.0,9.0) (4.0,6.0)
lft and rgt: (2.0,7.0) (6.0,7.0)
(4.0,6.0) (3.5,5.5)
lft and rgt: (2.0,7.0) (5.0,4.0)
(1.5,5.5) (3.5,5.5)
lft and rgt: (2.0,4.0) (2.0,7.0)
(3.5,5.5) (3.5,2.0)
lft and rgt: (2.0,4.0) (5.0,4.0)
